app/server/api/static/Find Results

```diff
@@ -0,0 +1,33 @@
+Searching 77 files for "isArray"
```

Comment:
Looks like this was committed by accident

Reply:
Removed